Frontend/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/node": "^6.0.45",
     "angular-in-memory-web-api": "~0.2.2",
     "bootstrap": "3.3.7",
-    "angular2-ladda": "^1.0.6",
+    "angular2-ladda": "1.0.7",
```

Comment:
https://docs.npmjs.com/getting-started/semantic-versioning
It should be all freeze or none, not this halfway point we are at.